src/spesh/optimize.c

```diff
@@ -929,19 +929,21 @@ static void optimize_repr_op(MVMThreadContext *tc, MVMSpeshGraph *g, MVMSpeshBB
 /* smrt_strify and smrt_numify can turn into unboxes, but at least
```

Comment:
You could mention smrt_intify here

Reply:
Yeah, updated a couple other comments also and force pushed.